openstreetmap-cgimap.1

```diff
@@ -55,6 +56,10 @@ Write pid to \fIPIDFILE\fR.
 .BR \-\-logfile =\fILOGFILE\fR
 File to write log messages to.
 .TP
+.BR \-\-configfile =\fICONFIGFILE\fR
+File to read configuration values from.
+Default is '/usr/local/etc/cgimap/cgimap.xml'.
+.TP
```

Comment:
Added `--configfile` command line argument, description, and default value to `man` page.